### PR TITLE
Fix a few things...

### PR DESCRIPTION
**Pull request by kitemongerer**

### CODE REVIEW

File: content/post/headless-ui-testing-with-go-agouti-and-chrome.md [hunk 1]
```diff
@@ -223,12 +226,12 @@ Let's now enter the container and run test scripts as if we were the concourse t
     go get github.com/sclevine/agouti
     service dbus restart
     cd $GOPATH/src/website
-    xvfb-run TEST_ENV=CI ginkgo -v
+    xvfb-run ginkgo -v
```
Comment (jadekler) [inline content/post/headless-ui-testing-with-go-agouti-and-chrome.md]: Why did you remove TEST_ENV? Our `website_suite_test.go` uses this variable. Is it being exported somewhere else?

**Comment by jadekler** [comment]
👍 thank you!

File: content/post/headless-ui-testing-with-go-agouti-and-chrome.md (hunk 1 above)
Comment (kitemongerer) [inline content/post/headless-ui-testing-with-go-agouti-and-chrome.md]: Sorry, I didn't see that. I thought you just forgot to take it out of our command
